File: AkitaModel/ResultEntry.cs
namespace Skyware.Lis.AkitaModel;

/// <summary>
/// Allowed result entry.
/// </summary>
public class ResultEntry
{

    /// <summary>
    /// Allowed value of the result.
    /// </summary>
    public string Value { get; set; }

    /// <summary>
    /// Corresponding flag for this value.
    /// </summary>
    public FlagLevels FlagLevel { get; set; } = FlagLevels.NONE;

}

File: AkitaMotelTS/src/ResultEntry.ts
/**
 * Skyware.Lis.AkitaModel
 */

import { FlagLevels } from './FlagLevels';

/**
 * Allowed result entry.
 */
export class ResultEntry {
  /**
   * Allowed value of the result.
   */
  value?: string;

  /**
   * Corresponding flag for this value.
   */
  flagLevel: FlagLevels = FlagLevels.NONE;
}
